moped-database/migrations/1649874244000_alter_project_list_view/up.sql

```diff
@@ -0,0 +1,46 @@
+drop view project_list_view;
+
+CREATE VIEW public.project_list_view AS
+ SELECT mp.project_uuid,
+    mp.project_id,
+    mp.project_name,
+    mp.project_description,
+    mp.project_description_public,
+    mp.ecapris_subproject_id,
+    mp.project_importance,
+    mp.project_order,
+    mp.current_status,
+    mp.timeline_id,
+    mp.current_phase,
+    mp.end_date,
+    mp.project_length,
+    mp.fiscal_year,
+    mp.capitally_funded,
+    mp.project_priority,
+    mp.date_added,
+    mp.added_by,
+    mp.is_retired,
+    mp.milestone_id,
+    mp.task_order,
+    COALESCE(mp.status_id, 0) AS status_id,
+    string_agg(DISTINCT concat(mu.first_name, ' ', mu.last_name, ':', mpr.project_role_name), ','::text) AS project_team_members,
+    mp.updated_at,
+    me.entity_name AS project_sponsor,
+    string_agg(DISTINCT me2.entity_name, ', '::text) AS project_partner,
+        CASE
+            WHEN ((mp.status_id = 0) OR (mp.status_id IS NULL)) THEN NULL::text
+            WHEN (mp.status_id = 1) THEN mp.current_phase
+            ELSE mp.current_status
+        END AS status_name,
+    string_agg(task_order_filter->>'display_name', ',') as task_order_name,
+    mp.contractor,
+    mp.purchase_order_number
+   FROM ((((((public.moped_project mp
```

Comment:
`((((((`

This `FROM` clause is using a lot of parenthetical grouping to make explicit the default order and binding of the `JOIN` operators. In my opinion, unless you're changing the order of operations, it's more clear to use include the precedence operators. (keywords? syntax? I'm not sure what you call what parentheses are.) 

I think these could be removed to increase clarity and not modify the way the query works.

Reply:
removed them and does not seem to affect the query behavior one bit. thank you